pesde.toml

```
@@ -1,37 +1,36 @@
-name = "0x5eal/unzip"
-version = "0.1.3"
-description = "Unzip implementation in pure Luau"
-authors = ["Erica Marigold <hi@devcomp.xyz>"]
-repository = "https://github.com/0x5eal/luau-unzip"
-license = "MIT"
-includes = [
-	"README.md",
-	"LICENSE.md",
-	"pesde.toml",
-	"lib/**/*.luau",
-	"!tests/**/*",
-  "docs/**/*.md"
-]
-
-[engines]
-pesde = "=0.6.0"
-lune = "=0.8.9"
-
-[scripts]
-tests = ".lune/tests/init.luau"
-fmt = ".lune/fmt.luau"
-typecheck = ".lune/typecheck.luau"
-generate_invalid_version_zip = "scripts/generate_invalid_version_zip.luau"
-
-[target]
-environment = "luau"
-lib = "lib/init.luau"
-
-[indices]
-default = "https://github.com/pesde-pkg/index"
-
-[dev_dependencies]
-stylua = { name = "pesde/stylua", version = "=2.0.2", target = "lune" }
-frktest = { name = "itsfrank/frktest", version = "^0.0.2", target = "lune" }
-asciitable = { name = "kimpure/asciitable", version = "^0.2.1" }
-luau-lsp = { name = "pesde/luau_lsp", version = "=1.39.2", target = "lune" }
+name = "0x5eal/unzip"
+version = "0.1.3"
+description = "Unzip implementation in pure Luau"
+authors = ["Erica Marigold <hi@devcomp.xyz>"]
+repository = "https://github.com/0x5eal/luau-unzip"
+license = "MIT"
+includes = [
+	"README.md",
+	"LICENSE.md",
+	"pesde.toml",
+	"lib/**/*.luau",
+	"!tests/**/*",
+  	"docs/**/*.md"
+]
+
+[engines]
+pesde = "^0.7.0"

```

For reproducibility via nix, we always pin our tooling versions to a fixed one. 

```suggestion
pesde = "=0.7.0"
```